### PR TITLE
Extract a transformer method for finer control of the formats and fields sent to logstash

### PR DESCRIPTION
**Pull request by JeffAtDeere**
Introduced a new option `transformer` as a function so that a client can exactly control what is sent to logstash.

### CODE REVIEW

**Comment by JeffAtDeere** [comment]
@djMax the build failed in a dep due to node 0.12 and the use of `const`. I assume a master build would fail as well so I hope you consider merging this.

File: lib/winston-logstash.js [hunk 1]
```diff
@@ -50,6 +50,7 @@ var Logstash = exports.Logstash = function (options) {
   this.strip_colors        = options.strip_colors || false;
   this.label               = options.label || this.node_name;
   this.meta_defaults       = options.meta || {};
+  this.transformer         = options.transformer || defaultTransformer;
```
Comment (djMax) [inline lib/winston-logstash.js]: I know this is super nitpicky, but could you change the name of this to "transform"? Seems more in line with naming conventions I've seen.

Comment (JeffAtDeere) [reply lib/winston-logstash.js]: done

**Comment by JeffAtDeere** [comment]
Any hope of a release or is this blocked by #47 ?

**Pull request merged**